linux_os/guide/system/permissions/files/permissions_within_important_dirs/file_groupownership_library_dirs/tests/correct_permissions.pass.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+for LIBDIR in /lib/ /lib64/ /usr/lib/ /usr/lib64/
+do
+   if [[ ! -f $LIBDIR ]]
+   then
+     touch $LIBDIR/LIBDIR
```

Comment:
Maybe this might be clearer?
```suggestion
     touch $LIBDIR/testfile
```